libs/blocks/section-metadata/section-metadata.css

```diff
@@ -222,6 +236,16 @@ main > .section[class*='-up'] > .content {
   }
 }
 
+@media screen and (min-width: 768px) and (max-width: 1199px) {
```

Comment:
Shouldn't this be `min-width: 600px`? `768` isn't one of our breakpoint as far as I can remember

Reply:
@ryanmparrish As per the XD shared the layout will have a two column grid for media screens 768px to 1199px and below 768px it is aligned to have a one column layout only hence we have used the explicit values.